Dockerfile

```diff
@@ -1,40 +1,50 @@
-FROM osrf/ros:noetic-desktop
+FROM ros:noetic-ros-core
 
 # Use bash instead of sh
 SHELL ["/bin/bash", "-c"]
 
 # Update Ubuntu Software repository
 RUN apt update && \
-    apt install -y \
+    apt install --no-install-recommends -y \
         git \
+        build-essential \
         python3-dev \
-        python3-pip
-
-RUN pip3 install \
-        conan \
+        python3-pip && \
+    pip3 install \
```

Comment:
An indentation fix is needed in this line.
We can additionally sort installed packages alphabetically.

Reply:
That is another command, I don't see a point in indenting it.

Reply:
My mistake, didn't catch it 👍🏼